src/SimpleBoundaryValueDiffEq.jl

```diff
@@ -1,5 +1,19 @@
 module SimpleBoundaryValueDiffEq
 
-# Write your package code here.
+using DiffEqBase, DifferentialEquations, NonlinearSolve, Plots
 
+# Only for out-of-place BVP, in place needs to be converted to out-of-place
```

Comment:
No don't worry about inplace, converting inplace to an OOP is almost always a terrible idea.